src/generic/modal-dropzone/messages.js

```diff
@@ -21,6 +21,10 @@ const messages = defineMessages({
     id: 'course-authoring.certificates.modal-dropzone.upload.modal',
     defaultMessage: 'Upload',
   },
+  uploadImageDropzoneInvalidSizeMore: {
+    id: 'course-authoring.certificates.modal-dropzone.validation.invalid-size-more',
```

Comment:
[question] is it a new translation, do we need to add it to i18n folder?

Reply:
Yes, thank you!